drivers/usbhost/Kconfig

```diff
@@ -10,6 +10,7 @@ config USBHOST_HAVE_ASYNCH
 menuconfig USBHOST
 	bool "USB Host Driver Support"
 	default n
+	depends on ALLOW_BSD_COMPONENTS
```

Comment:
look like usbhost framework already Apache copyright. Should we put ALLOW_BSD_COMPONENTS to the individual driver instead?

Reply:
https://github.com/jerpelea/incubator-nuttx/blob/master/drivers/usbhost/Make.defs#L21

Reply:
Done: https://github.com/apache/incubator-nuttx/pull/5578

Reply:
> https://github.com/jerpelea/incubator-nuttx/blob/master/drivers/usbhost/Make.defs#L21
> 
> I can add a new option for the HID Parser and then ifdef it Does this work for you ?

hid_parser.c is only used by usbhost_hidkbd.c and usbhost_hidmouse.c, it's better to move the dependence to them.